searx/settings.yml

```
@@ -1214,10 +1214,10 @@ engines:
     shortcut: pd
     search_url: https://pub.dev/packages?q={query}&page={pageno}
     paging: true
-    results_xpath: /html/body/main/div/div[@class="search-results"]/div[@class="packages"]/div
+    results_xpath: //div[@class="packages-item"]
```

```suggestion
    results_xpath: //div[contains(@class,"packages-item")]
```
This should make it more robust in case they add an additional class to it.

---

nice, thank you!